core/src/main/java/ai/timefold/solver/core/impl/domain/common/accessor/gizmo/GizmoMemberAccessorImplementor.java

```diff
@@ -246,9 +254,8 @@ private static void createGetName(ClassCreator classCreator, GizmoMemberInfo mem
 
         // If it is a method, assert that it has the required
         // properties
-        memberInfo.getDescriptor().whenIsMethod(method -> {
-            assertIsGoodMethod(method, memberInfo.getAnnotationClass());
-        });
+        memberInfo.getDescriptor().whenIsMethod(
+                method -> assertIsGoodMethod(method, memberInfo.isRequiredReturnType(), memberInfo.getAnnotationClass()));
```

Comment:
It's a race to the finish line now. :-) We're going to have conflicts, you and I.
Nevermind, we'll figure it out when the time comes.

Reply:
Yeah, I think so :-)